### ISSUE TITLE
CMake bug on symbolically linked lib directory

### ISSUE THREAD

**Issue by qnzhou** [CGAL/cgal#1632]
CGAL version: 4.9, download from http://www.cgal.org/2016/09/19/cgal-49/

CMake script break down when the $CGAL_DIR points to a symbolic linked dir on linux.

E.g. Assuming libCGAL.so is in `/usr/local/lib64`, and `/usr/local/lib` is a symbolic link to it.

The automatically generated file `/usr/local/lib/CGAL/CGALConfig.cmake` contains the following lines:

```
get_filename_component(CGAL_CONFIG_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
string(REPLACE "lib64/CGAL" "" CGAL_INSTALL_PREFIX "${CGAL_CONFIG_DIR}")
```

Which produced the wrong `CGAL_INSTALL_PREFIX`.

**Comment by lrineau** [comment]
@sloriot That is related to #1183, merged into CGAL-4.9. I think that should be fixed for CGAL-4.9.1.

**Comment by lrineau** [comment]
@qnzhou Can you please try the PR #1637? It should solve your issue.

**Comment by lrineau** [comment]
Fixed by #1637.

@qnzhou Let us know if you still have similar  issues.
